.gitignore

```diff
@@ -1,2 +1,3 @@
 /tmp
 /test/functional/web/tmp
+.idea/*
```

Comment:
small 💅🏼 - you could add this **your** local `.gitignore`, and all the projects using JetBrains' tools won't dirty the projects' `.gitignore`